lib/perl/Genome/Qc/Tool.pm

```diff
@@ -25,7 +25,15 @@ class Genome::Qc::Tool {
 
 sub cmd_line {
     my $self = shift;
-    die $self->error_message("Abstract method run must be overriden by subclass");
+    my $cmd = $self->gmt_class->create($self->gmt_params);
+    return $cmd->build_cmdline_list;
```

Comment:
I agree with you about the code in cmd_line and will move it to Picard.pm

I was hoping that output_file_accessor could become a more global solution via the logic in Result.pm for any tool that needs to parse an output_file for metrics. Right now we really only have Picard tools that fit that criteria.